cardigan/stories/components/InfoBlock.js

```diff
@@ -0,0 +1,30 @@
+import { storiesOf } from '@storybook/react';
+import InfoBlock from '../../../common/views/components/InfoBlock/InfoBlock';
+import Readme from '../../../common/views/components/InfoBlock/README.md';
+
+const InfoBlockExample = () => {
+  return (
+    <InfoBlock
+      heading={'Book your ticket'}
```

Comment:
Where do we use heading and where do we use title?

Reply:
You could be right. I suppose I was going on feel and felt like I would describe the thing at the top of these blocks as 'headings', but I think you're probably right about us previously having tried to stick to one. I'll change this to `title`.